base/abstractarray.jl

```diff
@@ -1829,6 +1829,22 @@ dimensions for every new input array and putting zero blocks elsewhere. For exam
 `cat(matrices...; dims=(1,2))` builds a block diagonal matrix, i.e. a block matrix with
 `matrices[1]`, `matrices[2]`, ... as diagonal blocks and matching zero blocks away from the
 diagonal.
+
```

Comment:
Maybe it should also link to its friends?
```suggestion

See also [`hcat`](@ref), [`vcat`](@ref), [`hvcat`](@ref), [`repeat`](@ref).

```
(I would include `diagm` except I'm not sure the link will work.)